wremnants/include/theoryTools.h

```diff
@@ -139,6 +139,39 @@ helicity_scale_tensor_t makeHelicityMomentScaleTensor(const CSVars &csvars, cons
 
   return original_weight*scale_tensor.reshape(reshapescale).broadcast(broadcasthelicities)*moments.broadcast(broadcastscales);
 
+}
+
+template <std::size_t nSize>
+Eigen::TensorFixedSize<double, Eigen::Sizes<9, nSize>> makePDFMomentScaleTensor(const CSVars &csvars, const  Eigen::TensorFixedSize<double, Eigen::Sizes<nSize>>&pdf_tensor, double original_weight = 1.0) {
```

Comment:
Should recycle code from the non-PDF function. Similar implementation in #310 so discuss with @emanca how to implement this